webxrlayers-1.bs

```diff
@@ -0,0 +1,730 @@
+ 1. during {{XRSession/requestAnimationFrame()}} for webgl layers, draw content each gl layer
+
+</section>
+
+Model {#model}
+======
+
+In order for the applications to signal their interest in using layers during a session, 
+the session must be requested with appropriate [=feature descriptor=]. The string <dfn>layers</dfn> is introduced 
+by this module as a new valid feature descriptor for WebXR Layers feature.
+
+<div class="example">
+The following code requests layers as an optional feature.
+
+<pre highlight="js">
+navigator.xr.requestSession('immersive-vr', {
+    optionalFeatures: ['layers']
+  }
+</pre>
+</div>
+
+Layers are only supported for XRSessions created with XRSessionMode of {{XRSessionMode/"immersive-vr"}} 
+or {{XRSessionMode/"immersive-ar"}}. {{XRSessionMode/"inline"}} sessions must not support layers.
+
+Layer types {#xrlayertypes}
+===========
+
+Mono and stereo layers {#monovsstereo}
+----------------------
+A stereo layer must supply a different view to render to for each eye. Stereo layers should be refreshed 
+close to the device's native frame rate.
+
+A mono layers must supply a single view which is shown to each eye. There is no requirement to refresh mono 
+layers repeatedly.
+
+The [=XR Compositor=] must ensure that layers are presented correctly in stereo to the observer.
+
+XRLayer {#xrlayertype}
+-------
+Every layer except XRWebGLLayer is derived from XRLayer.
+XRLayer defines a set of common attributes and behaviors acress all layer types.
+
+<pre class="idl">
+interface XRLayer {
+  readonly attribute unsigned long pixelWidth;
+  readonly attribute unsigned long pixelHeight;
+
+  attribute boolean blendTextureSourceAlpha;
+  attribute boolean chromaticAberrationCorrection;
+
+  void destroy();
+};
+</pre>
+
+The <dfn attribute for="XRLayer">pixelWidth</dfn> and <dfn attribute for="XRLayer">pixelHeight</dfn> attributes 
+return the width and height of the GL attachments, respectively. If there are no attachments, they will return 0.
+
+The <dfn attribute for="XRLayer">blendTextureSourceAlpha</dfn> attribute enables the layer’s texture alpha channel.
+
+The <dfn attribute for="XRLayer">chromaticAberrationCorrection</dfn> attribute enables optical chromatic 
+aberration correction for the layer when not done by default.
+
+{{destroy()}} will delete the underlying attachments. If there are no attachments, this function does nothing.
+
+Each {{XRLayer}} has a <dfn for="XRLayer">context</dfn> object which is an instance 
+of either a {{WebGLRenderingContext}} or a {{WebGL2RenderingContext}}.
+
+Each {{XRLayer}} has an associated <dfn for="XRLayer">session</dfn>, which is the 
+{{XRSession}} it was created with.
+
+XRProjectionLayer {#xrprojectionlayertype}
+-----------------
+A XRProjectionLayer is a layer that fills the entire view of the observer.
+Projection layer must always run stereo and should be refreshed close to the device's native frame rate.
```

Comment:
> Projection layer must always run stereo

That's not accurate for phone AR or similar form factors. Would be more correct to say they must have a separate sub image for each XRView or something similar.

Reply:
I just removed this. 
Since all layers can now be now be mono or stereo, should we move the stereo attribute to XRLayer?